docs/ARMInstallationManagedIdentity.md

```diff
@@ -0,0 +1,59 @@
+|**Repository Branch**|Source code branch to deploy.
+|**Job Window Unit**|The time period to collect events before sending them to the FHIR server.
+|**Job Window Magnitude**|The magnitude of time period to collect events before sending them to the FHIR server.
+|**Streaming Units**|Number of Streaming Units for the Stream Analytics job processing device events. For more information see [understanding Streaming Units](https://docs.microsoft.com/en-us/azure/stream-analytics/stream-analytics-streaming-unit-consumption) in the Stream Analytics documentation.
+|**Throughput Units**| The throughput units reserved for the Event Hubs created. For more information see [Throughput units FAQ](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-faq#throughput-units) in the Event Hubs documentation.
+|**App Service Plan SKU**|The app service plan tier to use for hosting the required Azure Functions.
+|**Resource Location**|The location of the deployed resources.
+|**FHIR Service URL**|URL of the FHIR server that IoMT data will be written to.
+|**Resource Identity Service Type**|Configures how patient, device, and other FHIR resource identities are resolved from the ingested data stream. The different supported modes are further documented below.
+|**Default Device Identifier System**|Default system to use when searching for device identities. If empty system is not used in the search.
+
+### Resource Identity Service Type
+**Note** all identity look ups are cached once resolved to decrease load on the FHIR server.  If you plan on reusing devices with multiple patients it is advised you create a *virtual device* resource that is specific to the patient and the virtual device identifier is what is sent in the message payload. The virtual device can be linked to the actual device resource as a parent.
+
+|Type|Behavior
+|---|---
+|**R4DeviceAndPatientLookupIdentityService**|Default setting.  Device identifier from ingested messages is retrieved from the FHIR server. Patient is expected to be linked to the device.
+|**R4DeviceAndPatientCreateIdentityService**|System attempts to retrieve the device identifier and associated patient from the FHIR server. If either isn't found a shell resource with just the identity will be created. Requires a patient identifier be mapped in the device content configuration template.
+|**R4DeviceAndPatientWithEncounterLookupIdentityService**|Like the first setting but allows you to include an encounter identifier with the message to associate with the device/patient.  The encounter is looked up during processing and any observations created are linked to the encounter. The association here is assumed to be one encounter per device.
+
+## Post Deployment
+After the ARM template is successfully deployed, add the Managed Identity ID output by the ARM deployment to the Allowed Object IDs on the Authentication page of your Azure API for FHIR.
+
```

Comment:
I know it's redundant, but I feel just a quick blurb here about using the Azure Active Directory as the identity provider for the Azure API for FHIR will reduce the chance of confusion. In fact we should point our that changing the Authority on the Authentication page of your Azure API for FHIR will cause the connector to error with 401s.